src/styles/borders.js

```diff
@@ -16,7 +16,7 @@ border.default = css`
 
 border.buildCircle = size => `
   ${position.size(size)};
-  border-radius: ${size / 2};
+  borderRadius: ${size / 2};
```

Comment:
This is still inside a string — should this be changed?

Reply:
There is object version too. Most of these have object version.

Reply:
I mean that this property name uses camel case even though it's inside a string. Shouldn't this stay as `border-radius`?